src/tests/dispatch-payload.test.ts

```diff
@@ -195,10 +212,11 @@ Deno.test("DispatchPayload with unhandled events", async (t) => {
       mock.assertSpyCalls(unhandledEventSpy, 0);
       mock.assertSpyCall(defaultSpy, 0, {
         args: [{
+          body: payload.body,
           event: payload.body.event,
           env: payload.context.variables,
           team_id: payload.context.team_id,
-          enterprise_id: payload.body.enterprise_id,
+          enterprise_id: "",
```

Comment:
To get slightly higher test coverage, we could write tests that specifically craft a payload with the appropriate `enterprise_id` properties being present (to simulate execution in a grid environment). Our tests currently do not do that, which is why the codecov warnings exist in `dispatch-payload.ts`' `extractBaseHandlerArgsFromPayload` method.